src/main/java/no/nav/familie/inntektsmelding/integrasjoner/person/PersonTjeneste.java

```diff
@@ -51,30 +50,14 @@ public PersonInfo hentPersonInfo(AktørIdEntitet aktørId, Ytelsetype ytelseType
             () -> new IllegalStateException("Finner ikke personnummer for id " + aktørId));
 
         var person = pdlKlient.hentPerson(utledYtelse(ytelseType), request, projection);
-
-        return new PersonInfo(mapNavn(person), personIdent, aktørId, mapFødselsdato(person));
-    }
-
-    private String mapNavn(Person person) {
-        return person.getNavn()
-            .stream()
-            .map(PersonTjeneste::mapNavn)
-            .flatMap(Optional::stream)
-            .findFirst()
-            .orElseGet(() -> Environment.current().isProd() ? null : "Navnløs i Folkeregister");
+        var navn = person.getNavn().getFirst();
```

Comment:
Kan person.getNavn() være null?

Reply:
Aner ikke 🤔 Det er en liste så synes jo det burde være en tom liste isåfall. Men jeg kjenner ikke til PDL så godt, så vet ikke. Hvis det er en tom liste så kaster jo getFirst en exception, men det er kanskje greit? Tar gjerne innspill til feilhåndtering her

Reply:
glem min kommentar -ser nå at den ikke kan være null